File: Beecrowd/Beginner/1000 - 1099/1021 - Banknotes and Coins/C++/1021_BanknotesandCoins.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    double amount;
    cin >> amount;

    int notes[6] = {100, 50, 20, 10, 5, 2};
    int coins[6] = {100, 50, 25, 10, 5, 1};

    cout << "NOTAS:" << endl;
    for (int i = 0; i < 6; i++) {
        int note = notes[i];
        int quantity = amount / note;
        amount -= quantity * note;
        cout << quantity << " nota(s) de R$ " << fixed << setprecision(2) << note / 1.00 << endl;
    }

    amount *= 100;

    cout << "MOEDAS:" << endl;
    for (int i = 0; i < 6; i++) {
        int coin = coins[i];
        int quantity = amount / coin;
        amount -= quantity * coin;
        cout << quantity << " moeda(s) de R$ " << fixed << setprecision(2) << coin / 100.0 << endl;
    }
    
    return 0;
}

File: Beecrowd/1 - Beginner/1000 - 1099/1021 - Banknotes and Coins/1021_BanknotesandCoins.py
amount = float(input())

notes = [100, 50, 20, 10, 5, 2]
coins = [100, 50, 25, 10, 5, 1]

print("NOTAS:")
for note in notes:
    quantity = int(amount / note)
    amount -= quantity * note
    print(f"{quantity} nota(s) de R$ {note / 1.00:.2f}")

amount *= 100

print("MOEDAS:")
for coin in coins:
    quantity = int(amount / coin)
    amount -= quantity * coin
    print(f"{quantity} moeda(s) de R$ {coin / 100.0:.2f}")